testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViewsLegacy.java

```diff
@@ -15,33 +15,27 @@
 
 import io.trino.tempto.BeforeTestWithContext;
 import io.trino.tempto.Requires;
-import io.trino.tempto.assertions.QueryAssert;
 import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
 import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableOrdersTable;
 import io.trino.tempto.query.QueryExecutor;
 import org.testng.annotations.Test;
 
 import java.math.BigDecimal;
-import java.sql.Date;
-import java.time.LocalDate;
-import java.util.function.Consumer;
 
 import static io.trino.tempto.assertions.QueryAssert.Row.row;
 import static io.trino.tempto.assertions.QueryAssert.assertThat;
 import static io.trino.tempto.query.QueryExecutor.query;
 import static io.trino.tests.TestGroups.HIVE_VIEWS;
-import static io.trino.tests.utils.QueryExecutors.connectToPresto;
 import static io.trino.tests.utils.QueryExecutors.onHive;
 import static io.trino.tests.utils.QueryExecutors.onPresto;
-import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @Requires({
         ImmutableNationTable.class,
         ImmutableOrdersTable.class,
 })
 public class TestHiveViewsLegacy
-        extends HiveProductTest
+        extends TestHiveViews
```

Comment:
can we make an abstract class and avoid a test extending some other test?

Reply:
Sure :)